docs/features/network-security-controls/network-policy.md

```diff
@@ -58,7 +178,107 @@ spec:
   policyTypes:
   - Egress
 ```
-**OVN-Implementation:**
+
+#### **Selector Based Denial**
+
+We can extend the default denial behavior by using the `podSelector` field to selectively block all traffic from/to certain pods, in this case blocking all unicast traffic from reaching apps labeled with demo.
```

Comment:
I am not sure if it can be considered the extended behaviour, may be instead just say that netwokr policy doesn't necessarily applied to the whole namespace and you can select pods that need to be isolated via selector?